tests/js/setup.js

```diff
@@ -26,6 +26,11 @@ if (!SVGElement.prototype.getTotalLength) {
 
 /**
  * Enzyme configuration
+ *
+ * TODO(epurkhiser): We're using @wojtekmaj's react-17 enzyme adapter, until
+ * the offical adapter has been released.
+ *
+ * https://github.com/enzymejs/enzyme/issues/2429
```

Comment:
Is enzyme often lagging behind react something we should be taking into consideration as a reason to consider React Testing Library more? From [this comment](https://github.com/enzymejs/enzyme/issues/2429#issuecomment-817750268) it sounds like react-testing-library doesn't use react internals and is easier to upgrade with.

Reply:
I think this is a pretty strong argument for adopting React Testing Library

Reply:
Agree!!!!